pkg/cloudprovider/providers/aws/aws_test.go

```diff
@@ -587,9 +589,10 @@ func TestNodeAddresses(t *testing.T) {
 	// (we test that this produces an error)
 	var instance0 ec2.Instance
 	var instance1 ec2.Instance
+	var instance2 ec2.Instance
 
 	//0
-	instance0.InstanceId = aws.String("instance-same")
+	instance0.InstanceId = aws.String("i-self")
```

Comment:
Have to use this instance id so that it matches in the test fake DescribeInstances call and `getSelfAWSInstance` doesn't return an error.